### PR TITLE
kristoffer first dag

### PR DESCRIPTION
**Pull request by rodvei**
my first dag

### CODE REVIEW

**Comment by fredrikuhl** [comment]
looks good

**Review by mats-bb — APPROVED** [review]
dlgmt

File: dags/kristoffers_dag.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+default_args = {
+    "owner": "Kristoffer",
+    "retries": 2,
+    "retry_delay": datetime.timedelta(minutes=2),
+    "start_date": datetime.datetime(2023, 2, 1),
+}
+
```
Comment (Aliti-Coding) [inline dags/kristoffers_dag.py:12]: do you really need space here?

**Review by IngridMorch — COMMENTED** [review]
lgtm

**Review by drimepal — APPROVED** [review]
lgtm

**Review by Chris-RJ-Larsen — APPROVED** [review]
lgtm

**Review by GlenRoger — APPROVED** [review]
lgtm

**Pull request merged**